pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -82,27 +82,30 @@ func (configgen *ConfigGeneratorImpl) BuildClusters(proxy *model.Proxy, push *mo
 	switch proxy.Type {
 	case model.SidecarProxy:
 		// Setup outbound clusters
-		outboundPatcher := clusterPatcher{envoyFilterPatches, networking.EnvoyFilter_SIDECAR_OUTBOUND}
+		outboundPatcher := clusterPatcher{efw: envoyFilterPatches, pctx: networking.EnvoyFilter_SIDECAR_OUTBOUND, proxy: proxy}
```

Comment:
`proxy` does not need to be passed here and elsewhere?

Reply:
yeah., Good catch. Forgot to remove it from there. Fixed now. PTAL